tools/ci_download_clickhouse

```diff
@@ -7,112 +7,136 @@
 #
 
 set -o pipefail
-set -o xtrace
```

Comment:
Are you removing this because it pollutes stderr from the new top-level install-prerequisites script?  Would it be reasonable to redirect stderr instead?

(Context: among the problems I've experienced a lot with bash scripts is that aside from being brittle, they're hard to debug.  xtrace makes it so much more likely we'll be able to debug problems the first time they happen without asking a user (who in this case is new to the whole system) having to guess what went wrong.)